linux/device.go

```diff
@@ -1077,25 +1077,27 @@ func RescanForCapacityUpdates() error {
 		return err
 	}
 
-	// reconfigure multipath maps to apply new size
-	args = []string{"reconfigure"}
-	_, _, err = util.ExecCommandOutput("multipathd", args)
```

Comment:
should we put a shorter timeout for reload in case the multipath -r is hung for some reason?